### ISSUE TITLE
Customizer: Unable to scroll when using an iPad Pro in landscape more

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#30647]
The user reported that they're unable to scroll in the Customizer, or in other parts of the Calypso UI when using their iPad Pro, in landscape mode, and with a smart keyboard.

#### Steps to reproduce
1. Open the Customizer in desktop view in either Safari or Firefox on the iPad
2. Open the custom CSS panel
3. Attempt to scroll up or down in the custom CSS panel
4. Tapping to scroll adds an insertion point, but doesn't enable scrolling

They also described a similar issue adding an image:

>One example is inserting an image. You get the image chooser in a pop up window. But you have to scroll way way down to get the select button. Except that almost always in landscape mode you can never scroll. If I flip the iPad into portrait then you can scroll. But only, if you restart this process. If you flip orientation in the middle of this WP JavaScript goes wonky , the screen is all dim and you’re hosed basically. Same scrolling problem exists in the custom CSS editor. That one is worse as the iPad hardware keyboard does not work at all in that css field. Very painful.

#### What I expected

Scrolling on the iPad Pro should behave like scrolling on other devices, where focus and scrolling is enabled using available inputs such as the touch screen or smart keyboard.

#### What happened instead

Scrolling doesn't work properly.

#### Browser / OS version

iPad Pro 2018 12.9” with Smart Keyboard.

iOS 12.1.3 

Firefox for iOS

#### Screenshot / Video

Please see 1782668-zen for links to videos demonstrating the issue.

#### Context / Source

User report in 10173431-hc and 1782668-zen

**Comment by pauljacobson** [comment]
It's also worth noting that the user is using the Twenty Nineteen theme.

**Comment by eduardozulian** [comment]
On top of that, the user shared another struggle:

> The selection issues also with the custom css field. Arrow keys on physical keyboard do not work so cannot select anything. iOS touch methods for selection also don’t work.

There is also a video on 1782668-zen for the CSS part.
